dojson/utils.py

```diff
@@ -96,8 +96,12 @@ def wrapper(self, key, values, **kwargs):
 
 def force_list(data):
     """Wrap data in list."""
-    if data is not None and not isinstance(data, (list, tuple, set)):
-        return (data,)
+    if data is None:
+        return []
```

Comment:
I understand @kaplun's point, the name `force_list` might be misleading. If I didn't work before with it and know its behaviour I would spect to get a list no matter the input.

If we stay with the current implementation IMHO at least we should update the docstring to reflect this special case.